packages/color-scale-block/src/components/ColorPickerFlyout.tsx

```diff
@@ -0,0 +1,83 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Let's change this and use the confirm button of the `Flyout` component, I'm sure the design team will insist on that.

So we don't fire the API `onSelect`, rather use the `onConfim` of the `Flyout` component and only fire if the confirm button is pressed.